src/api/client/resolvers.ts

```diff
@@ -57,29 +58,30 @@ export const queryNodeStitchingResolvers = (
       RemoveQueryNodeFollowsField,
     ]),
   },
-  Video: {
```

Comment:
I think this may still be needed though? For example, if you open video view, we only fetch a single video there, so without this that video won't contain the number of views. Currently what happens is that fetching "More videos from this channel" will actually end up populating that number but we shouldn't rely on that

Reply:
I think this is more problematic - if I bring this back and query videosConnection, `GetVideoViews` will be also called for every video.
Not sure if there is a way to not call that.

Reply:
Are you sure? I'd expect `GetVideoViews` to be called only if video is missing the `views` field

Reply:
Bringing back this resolver was not enough, I had to add one if statement. I think it should work just fine. Let me know what you think